espanol/exageracion/exageracion.md

```diff
@@ -0,0 +1,4 @@
+Después de sentir el dolor insoportable de los golpes de la escalera en sus rostros,
+empezaron a notar que sus estómagos intentaban de alguna forma comunicarse con ellos,
+GHHRHHRhhRh decia el estomago pero ellos no solo lo escuchaban sentían que de alguna forma sus estómagos cobraban vida propia,
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.